support/linting/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 syn.workspace = true
 quote.workspace = true
 proc-macro2.workspace = true
+procedural-fork = { version = "1.10.0-rc3", path = "../procedural-fork" }
```

Comment:
Shouldn't just the path attribute work alone, without the version?

Reply:
not if we publish to crates.io eventually, need both